core/src/main/java/io/grpc/internal/ServerImpl.java

```diff
@@ -574,7 +574,8 @@ public LogId getLogId() {
 
   @Override
   public ListenableFuture<ServerStats> getStats() {
-    ServerStats.Builder builder = new ServerStats.Builder();
+    ServerStats.Builder builder = new ServerStats.Builder()
```

Comment:
break line at highest semantic level, i.e. the `=`

Reply:
done